central/src/components/GameMaker.jsx

```diff
@@ -362,6 +361,7 @@ export default function GameMaker({ loading, game, newSave, editSave, gameId, cl
               question={gameNumber ? null : getGameById(games, gameId)?.questions[Number(createQuestionIndex) - 1]}
               updateQuestion={updateQuestion}
               cloneQuestion={cloneQuestion}
+              addQToGT={addQToGT}
```

Comment:
probably worth to have the full name, it's vague to know what this does without knowing the context

Reply:
fixed